libimage/filters.go

```diff
@@ -308,19 +309,20 @@ func filterReferences(r *Runtime, wantedReferenceMatches, unwantedReferenceMatch
 }
 
 // imageMatchesReferenceFilter returns true if an image matches the filter value given
```

Comment:
Please update the comment … and _maybe_ the function name.

Eg. do we even need the boolean return value?